cmd/kube-apiserver/app/patch_openshift.go

```diff
@@ -5,13 +5,10 @@ import (
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	clientgoinformers "k8s.io/client-go/informers"
 	"k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver"
-	"k8s.io/kubernetes/pkg/master"
```

Comment:
can this be squashed into another canonical commit introducing this code, potentially after splitting?

Reply:
Squashed into `UPSTREAM: <carry>: openshift-kube-apiserver: add kube-apiserver patches` along with `UPSTREAM: <carry> print merged flags at startup`.